js/src/components/modals/SemRushUpsellAlert.js

```diff
@@ -0,0 +1,42 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+
+/* Internal dependencies */
+import { Alert } from "@yoast/components";
+import { makeOutboundLink } from "@yoast/helpers";
+
+const PremiumLandingPageLink = makeOutboundLink();
+
+/**
+ * Creates the content for the Yoast SEO Premium upsell alert in SEMrush modal
+ *
+ * @returns {React.Element} The Yoast SEO Premium upsell alert
+ */
+const SemRushUpsellAlert = () => {
+
+	return (
+		<Alert type="info">
+			{
+				sprintf(
+					/* translators: %s : Expands to "Yoast SEO". */
+					__( "Would you like to be able to add these related keyphrases to the %s analysis so you can optimize your content even further?", "wordpress-seo" ),
+					"Yoast SEO"
```

Comment:
Running `grunt eslint` locally, it tells me:
`23:1   error  Line 23 exceeds the maximum line length of 150`

Reply:
I thought I had ESLint configured and working but it wasn't, now it is and I can see all these plus a couple more

Reply:
Also in RelatedKeyPhrasesModal.js:
18:61  error  Missing semicolon

Reply:
Thanks @enricobattocchi yeah, fixed. More importantly, we should investigate why Travis didn't run ESLint.